ras_backstage/resources/party/update_respondent_details.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from flask import request
+from flask_restplus import Resource, fields
+from structlog import wrap_logger
+
+from ras_backstage import party_api
+from ras_backstage.controllers import party_controller
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+respondent_details = party_api.model('RespondentDetails', {
+    'firstName': fields.String(required=True),
+    'lastName': fields.String(required=True),
+    'telephone': fields.String(required=True),
+})
+
+
+@party_api.route('/update-respondent-details')
+class UpdateRespondentDetails(Resource):
+    @staticmethod
+    @party_api.expect(respondent_details)
+    def post():
```

Comment:
This should probably be a put since we're updating existing details, same for the new endpoint in party service